lib/rules/web/admin_console/4.3/common_ui_elements.xyaml

```diff
@@ -88,11 +88,24 @@ click_overview_tab:
   params:
     tab_name: Overview
   action: click_tab
+click_yaml_tab:
+  params:
+    tab_name: YAML
+  action: click_tab
```

Comment:
rules copied for OCP-23967 to run on 4.3

Reply:
Thanks！